nixos/modules/services/networking/tailscale-auth.nix

```diff
@@ -0,0 +1,107 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
We should avoid this here

Reply:
I've replaced this with the `inherit (lib) ...` pattern.